src/commands/deployment/get.ts

```diff
@@ -2,95 +2,101 @@ import Table from "cli-table";
 import commander from "commander";
 import Deployment from "spektate/lib/Deployment";
 import AzureDevOpsPipeline from "spektate/lib/pipeline/AzureDevOpsPipeline";
-import IPipeline from "spektate/lib/pipeline/Pipeline";
 import { Config } from "../../config";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { isIntegerString } from "../../lib/validator";
 import { logger } from "../../logger";
-export let hldPipeline: IPipeline;
-export let clusterPipeline: IPipeline;
-export let srcPipeline: IPipeline;
+import { IConfigYaml } from "../../types";
+import decorator from "./get.decorator.json";
+
 /**
  * Output formats to display service details
  */
 export enum OUTPUT_FORMAT {
-  /**
-   * Normal format
-   */
-  NORMAL = 0,
+  NORMAL = 0, // normal format
+  WIDE = 1, // Wide table format
+  JSON = 2
+}
 
-  /**
-   * Wide table format
-   */
-  WIDE = 1,
+export interface IInitObject {
+  config: IConfigYaml;
+  clusterPipeline: AzureDevOpsPipeline;
+  hldPipeline: AzureDevOpsPipeline;
+  key: string;
+  srcPipeline: AzureDevOpsPipeline;
+}
+export interface ICommandOptions {
```

Comment:
nit: jsdoc for `ICommandOptions` and `IInitObject`

Reply:
accept